UPGRADING.md

```diff
@@ -218,7 +214,17 @@ From here on the doc will be focussed on an ODFE plugin as an example.
 
 See [anomaly-detection#35](https://github.com/opensearch-project/anomaly-detection/pull/35) for an example.
 
-### OpenSearch Dashboard Plugins
+##### Indices
+
+You can adopt one of the following stragegies.
+
+1. Don't change index names. For example, `.opendistro-anomaly-detectors` remains as such.
+
+2. Attempt to fallback to the old name. For example, if `".anomaly-detectors` is unavailable, look up `.opendistro-anomaly-detectors`.
+
+3. Check whether the old name exists, and alias it to the new name. For example, if `.opendistro-anomaly-detectors` exists, create a `.anomaly-detectors` alias.
```

Comment:
I would like to follow the convention of `.plugins-anomaly-detectors` just like the RestAPIs. What do you think?. We should make it consistent across all the items i.e APIs, indices, settings etc.